frontend/src/tests/components/Courses/CoursesTable.test.js

```diff
@@ -0,0 +1,83 @@
+import { render, waitFor, fireEvent } from "@testing-library/react";
+import CoursesTable from "main/components/Courses/CoursesTable";
+import {coursesFixtures} from 'fixtures/coursesFixtures'
+
+describe("CoursesTable tests", () => {
+
+    test("renders an empty table without crashing", () => {
+        render(
+            <CoursesTable courses={[]} />
+        );
+    });
+
+    test("renders a table with three rows without crashing", () => {
+        render(
+            <CoursesTable courses={coursesFixtures.threeCourses} />
+        );
+    });
+
+    test("CoursesTable is testId", async () => {
+        const {getByTestId } = render(
+            <CoursesTable courses={coursesFixtures.threeCourses} />
+        );
+        await waitFor( ()=> expect(getByTestId("CoursesTable-header-quarter")).toBeInTheDocument() );
+    });
+
+    test("click on a header and a sort caret should appear", async () => {
```

Comment:
IDK if it was implemented by you, but testing sort by description on threeCourses on storybook does not sort it correctly